Formula/l/literate-git.rb

```diff
@@ -54,6 +55,7 @@ class LiterateGit < Formula
     sha256 "491c8be9c040f5390f5bf44a5b07752bd07f56edf992381b05c701439eec10f6"
   end
 
+  # pygit2 must be kept below 1.15.0, until libgit2 1.8.0+ is merged: https://github.com/Homebrew/homebrew-core/pull/166767
```

Comment:
either remove this or move it to the top as it would affect the `brew update-python-resources` step